atlascope/core/job_types/nucleus_detection.py

```diff
@@ -63,7 +64,20 @@ def detect_nuclei(input_image):
         input_image,
     )
 
-    return additional_features.to_dict("records")
+    return (
+        additional_features.to_dict("records"),
+        np.vectorize(lambda x: 255 if x > 0 else 0)(im_nuclei_seg_mask).astype(np.uint8),
+    )
```

Comment:
Unless there's a specific reason to return a tuple, you're usually better off using a dict instead (those are more human-readable than a bare tuple).

Reply:
The reason may be the ease of destructuing the return value at the point of use; in that case, if you can add Python types to the function declaration, that would be very helpful.